app/src/androidTest/java/org/openobservatory/ooniprobe/test/WebConnectivityTest.java

```diff
@@ -0,0 +1,84 @@
+package org.openobservatory.ooniprobe.test;
+
+import android.util.Log;
+
+import com.google.gson.Gson;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openobservatory.engine.Engine;
+import org.openobservatory.engine.OONIMKTask;
+import org.openobservatory.ooniprobe.AbstractTest;
+import org.openobservatory.ooniprobe.model.jsonresult.EventResult;
+import org.openobservatory.ooniprobe.model.jsonresult.JsonResult;
+import org.openobservatory.ooniprobe.model.settings.Settings;
+
+import java.util.Collections;
+import java.util.concurrent.CountDownLatch;
+
+public class WebConnectivityTest extends AbstractTest {
+    private static final String MK_NAME = "WebConnectivity";
+    private static final String TAG = "integration-test";
+    private static final String UNUSED_KEY = "UNUSED_KEY";
+    private static final String CLIENT_URL = "https://ams-pg.ooni.org";
+
+    @Test
+    public void runWebConnectivity() {
+        final CountDownLatch signal = new CountDownLatch(1);
```

Comment:
This code is synchronous, so I think we don't need to use a countdown latch here.